packages/@groupby/elements-sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -136,7 +140,12 @@ export default class SaytDriverPlugin<P = Record> implements Plugin {
    * @param event Event that contains the Search API request payload.
    */
   fetchProductData(event: CustomEvent<SaytProductsRequestPayload>): void {
-    const { query, group, config } = event.detail;
+    const {
+      query,
+      group,
+      config,
+      origin,
+    } = event.detail;
```

Comment:
This could easily be one line, but I believe the linter complains if you import >= 4 modules on one line. Personally, I think that should just be a line length issue.